ffi/dotnet/Devolutions.IronRdp/src/Connection.cs

```diff
@@ -218,7 +239,7 @@ static async Task<NetworkStream> CreateTcpConnection(String servername, int port
         }
         catch (FormatException)
         {
-            IPHostEntry ipHostInfo = await Dns.GetHostEntryAsync(servername);
+            IPHostEntry ipHostInfo = await Dns.GetHostEntryAsync(servername).WaitAsync(TimeSpan.FromSeconds(5));
```

Comment:
question: Any specific reason for the 5 seconds timeout? We generally use 10 seconds elsewhere.

Reply:
updatedd